packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -55,13 +56,16 @@ export class RouterliciousDocumentServiceFactory implements IDocumentServiceFact
         private readonly tokenProvider: ITokenProvider,
         driverPolicies: Partial<IRouterliciousDriverPolicies> = {},
     ) {
+        // 5 days is the max allowed value per the IDocumentStorageServicePolicies.maximumCacheDurationMs policy
```

Comment:
Do you know if FRS uses this document service factory? I am just curious what would it take for this to implemented for FRS.

Reply:
AFAIK FRS uses r11s driver.  @znewton, right?